docs/demos/manipulation.md

```
@@ -75,6 +79,84 @@ manipulation techniques.
+    colcon build --symlink-install
+    ```
+
+#### 2. Running the demo
+
+!!! note Source the setup shell
+
+    Ensure ROS 2 is sourced on the host machine. Ensure that every command inside the docker container is run in a sourced shell using `source setup_shell.sh`.
+
+1. Launch the Robotic Arm Visualization on the host machine:
+
+    ```shell
```

markdownlint-cli2 (0.17.2)

Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)